### DISCUSSION TITLE
finer resolution for nudging midi sync

### DISCUSSION THREAD

**Discussion by Phexton** [SynthstromAudible/DelugeFirmware#1515] [Ideas]
in moment resolution is 1/96 per nudge step. @120BPM it’s around 20 milliseconds per step. That is a big time step if you work with other sequencers (master/slave) to have a good beat match together. If it will be possible to make that finer it would be great. maybe that 1/96 come from the midi time resolution 24ppqn, cause for a hole bar beat that are 96 pulse. So 1/96 note is equal 1 pulse?! A nudge resolution step per 1 millisecond or less would be much better ☺️
thanks for your great work!!!

**Comment by sapphire-arches** [comment]
I think this should improve if you increase the defaults>resolution setting. At least, you can nudge at finer resolution though I don't remember if we actually increase the MIDI out resolution.
